src/lib/components/ui/input/date-input.svelte

```diff
@@ -332,7 +326,7 @@
 		position: fixed;
 		top: 0;
 		left: 0;
-		z-index: 1;
+		z-index: 101;
```

Comment:
Do we even need this? Isn't 1 good enough and if not then some lower number than 100? :)

Reply:
no, because mode-button-container and menu-button-container in layout have z-index 100